tests/unit_tests/libs/controllers/test_predictor.py

```diff
@@ -484,3 +485,21 @@ def test_empty_column(self):
         )
 
         mdb.predict(when_data={'categorical_x': 0})
+
+    def test_output_class_distribution(self):
+        mdb = Predictor(name='test_output_class_distribution')
+        data = load_iris(as_frame=True)
+        df = data.data
+        df['target'] = data.target
+
+        mdb.learn(
+            from_data=df,
+            to_predict='target',
+            stop_training_in_x_seconds=1,
+            use_gpu=False,
+            advanced_args={'debug': True, 'output_class_distribution': True}
+        )
+
+        results = mdb.predict(when_data=data.data.iloc[[0]])
+        assert 'target_class_distribution' in results._data
+        assert np.isclose(np.sum(results._data['target_class_distribution']), 1)
```

Comment:
Add an assert of the nr of labels, i.e. the length of every member of the array under `target_class_distribution`, since this is presumably known for the training data and should be the same for every member.